MUST retire Connection IDs becoming stale (#3096)
quicwg/base-drafts: draft-ietf-quic-transport.md

An endpoint can request that its peer retire connection IDs by sending a
NEW_CONNECTION_ID frame with an increased Retire Prior To field.  Upon receipt,
the peer MUST retire the corresponding connection IDs and send corresponding
RETIRE_CONNECTION_ID frames.  Failing to retire the connection IDs within one

> PTO can cause packets to be delayed, lost, or cause the original endpoint to

Author: Thank you for the suggestions. I've added "approximately", and merged the other suggestion into the next paragraph. PTAL.

> An endpoint can request that its peer retire connection IDs by sending a

Reviewer: This isn't really a request any more, is it?
```suggestion
An endpoint can cause its peer to retire connection IDs by sending a
```
